nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/ErrorType.cs

```diff
@@ -0,0 +1,7 @@
+namespace NuGetUpdater.Core;
+
+public enum ErrorType
+{
+    None,
```

Comment:
Good call, I'll add that in a subsequent PR.  I'd like to merge what we have so we can get it into production, because this will deal with an issue I've seen in the logs in the last few minutes.

Reply:
Can you at least add a "TODO" comment int he code? That way it doesn't get forgotten about...